paint-timing/fcp-only/fcp-opacity-descendant.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<head>
+<title>Performance Paint Timing Test: FCP due to ancestor opacity</title>
+<style>
+    #main {
+        width: 100px;
+        height: 100px;
+        left: 0px;
+        position: relative;
+        top: 0;
+    }
+
+    #child {
+        opacity: 0;
+    }
+
+    #main.contentful #child {
```

Comment:
Hmm we seem to have multiple `#child` here, is that intended? or did you mean to apply contentful here too? Please take a look at the styles in this test

Reply:
Yes definitely! It's #child when it's descendant of #contentful.

Reply:
Oh doh sorry :) you're right